src/Console/ConfigurationResolver.php

```diff
@@ -161,6 +176,22 @@ public function getFormat()
     }
 
     /**
+     * @return ReportFactory
+     */
+    public function getReportFactory()
```

Comment:
what?
why?
how does ReportFactory is connected to resolving configuration?
ReportFactory is not aware of configuration.

Instead of ReportFactory it should return ready to use Report generator